calpip/Task1/Task 1-1.txt

```diff
@@ -1,21 +1,21 @@
 Dear Sir or Madam,
 
-My name is Wellington and I work voluntarily in a community organization that 
-makes a wonderful job for local children. I am writing to ask your company for
+My name is Wellington and I volunteer for a community organization that 
+provides a wonderful service for local children. I am writing to ask your company for
  a donation, which will be used to help us to organize an event next month.
 
-Although you probably have heard about us before, I would like to explain a 
-little bit about our charity. The organization which I help is called Angels 
-of the Music because we teach poor kids how to play a variety of musical 
+Although you have probably heard about us before, I would like to explain a 
+little bit about our charity. The organization I volunteer for is called Angels 
+of the Music. We teach kids from low income families how to play a variety of musical 
 instruments at no cost. We offer music classes every day in three different 
-parks of the city.
+parks of our city.
 
-Next month we are planning to realize a concert in which all the kids we have 
-taught could play on the main stage in downtown area. It will be a great night 
-full of good music, food and specially children’s happiness.
+Next month, we are planning to host a concert in which all the kids we have 
+taught to play can perform on the downtown area's main stage. We plan for it to be a great night 
+full of good music, food and especially children’s happiness.
```

Comment:
use commas where you would take a breath, as if you were speaking out loud


`realize` -> `host`
`Realize` relates more to dreams/fantasies and is 90% of the time not the right word to use, such as in this case where you describe an event you want to happen. Try to use another word when you feel like using "realize" in that context.


`It will be a great night` -> `We plan for it to be a great night`
This change is to allow you to use "however" in the next paragraph. It sounds less consistent to say:

> `it WILL be a great night...however it might not actually be`

compared to 

> `we hope it will be a great night...however it might not actually be`

where in the first sentence you are confident, then you doubt yourself, and in the second sentence you are more consistent with your message